### PR TITLE
fix(unlock-app, paywall-app, unlock-protocol-com): changed netlify API

### PR DESCRIPTION
**Pull request by julien51**
# Description

Adding back the `--dir` because for some reason netlify does not deploy it by default.
I also want to explore dynamic deploys so we could have server-side rendering...

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "devDependencies": {
     "eslint": "8.41.0",
     "eslint-plugin-prettier": "latest",
+    "eslint-plugin-vitest": "^0.2.6",
```
Comment (searchableguy) [inline package.json:20]:
```suggestion
    "eslint-plugin-vitest": "0.2.6",
```

**Review by searchableguy — APPROVED** [review]
LGTM

**Comment by julien51** [comment]
https://github.com/netlify/cli/issues/5809

**Comment by julien51** [comment]
I will hold since we are thinking about deploying on other platforms such as Cloudflare and/or Vercel!

**Pull request closed without merging**